src/models/files/ZosNetworkFile.js

```diff
@@ -0,0 +1,173 @@
+import _ from 'lodash'
+import { bytecodeDigest } from '../../utils/digest'
+import { Logger, FileSystem as fs } from 'zos-lib'
+
+const log = new Logger('ZosNetworkFile')
+
+export default class ZosNetworkFile {
+
+  constructor(packageFile, network, fileName) {
+    this.packageFile = packageFile
+    this.network = network
+    this.fileName = fileName
+
+    const defaults = this.packageFile.isLib() ? { contracts: {}, lib: true, frozen: false } : { contracts: {}, proxies: {} }
+    this.data = fs.parseJsonIfExists(this.fileName) || defaults
+  }
+
+  get app() {
+    return this.data.app
+  }
+
+  get appAddress() {
+    return this.app && this.app.address
+  }
+
+  get package() {
+    return this.data.package
+  }
+
+  get packageAddress() {
+    return this.package && this.package.address
+  }
+
+  get provider() {
+    return this.data.provider
+  }
+
+  get providerAddress() {
+    return this.provider && this.provider.address
+  }
+
+  get version() {
+    return this.data.version
+  }
+
+  get frozen() {
+    return this.data.frozen
+  }
+
+  get stdlib() {
+    return this.data.stdlib
+  }
+
+  get stdlibName() {
+    return this.stdlib.name
+  }
+
+  get stdlibVersion() {
+    return this.stdlib.version
+  }
+
+  get stdlibAddress() {
+    return this.stdlib.address
+  }
+
+  get proxies() {
+    return this.data.proxies
+  }
+
+  get contracts() {
+    return this.data.contracts
+  }
+
+  proxy(alias) {
```

Comment:
`proxy` in singular is misleading, as this returns an array

Reply:
Yes, I noticed that after pushing this PR and fixed it in a following one (see https://github.com/zeppelinos/zos-cli/pull/261/files#diff-a2f915840192bab0d44e9d30d58495d9R93)